src/index.js

```diff
@@ -1,3 +1,48 @@
-// eslint-disable-next-line no-console
-console.log('Hello! The current date and time is: ' + Date());
+var fs;
+var promises;
+var request;
 
+const meetups = [
```

Comment:
Consider moving this list into a separate file.

Reply:
Agreed, and the idea is that in the near short -> long term is that this list would be managed through the CMS we will expose (via REST API).